puppet/modules/redmine/manifests/init.pp

```diff
@@ -166,16 +166,24 @@
     content => template('redmine/redmine.service.erb'),
   }
 
+  anubis::instance { 'redmine':
```

Comment:
technically, this should be in the remine *profile* and the httpd config should also be there, but I didn't plan on a big refactoring today.

Reply:
or maybe, this IS the redmine profile? :)